be/src/vec/exprs/vectorized_fn_call.cpp

```diff
@@ -104,12 +104,38 @@ doris::Status VectorizedFnCall::execute(VExprContext* context, doris::vectorized
     size_t num_columns_without_result = block->columns();
     // prepare a column to save result
     block->insert({nullptr, _data_type, _expr_name});
+    if (_function->can_fast_execute()) {
```

Comment:
because in `fast_execute`  to build unique identifier of expr need get condition's value, other function maybe not have condition's value in arguments